src/refactor_nrepl/ns/pprint.clj

```diff
@@ -153,7 +153,7 @@
         forms (cond (and docstring? attrs?) (nthrest more 2)
                     (not (or docstring? attrs?)) more
                     :else (rest more))
-        ns-meta (:top-level-meta (meta ns-form))]
+        ns-meta (apply dissoc (:top-level-meta (meta ns-form)) (keys attrs?))]
```

Comment:
What does this accomplish, and does it have code coverage?

Reply:
Previously `:top-level-meta` did not contain attr-map of namespace, so `pprint` function assumed that all ns metadata is supplied as meta (`^{}`) and during printing put all contents of top level meta before namespace name (`(ns ^{:a 1} ns.name)`). Now that attr-map and ns-meta are merged, metadata from attr-map gets printed twice: first in ns meta, then in attr-map. So, if we have namespace like this
```clj
(ns ^:foo some.ns {:bar false})
```
it will be `pprint`ed like
```clj
(ns ^{:foo true, :bar false} some.ns {:bar false})
```
To prevent this from happening I changed it so that it excludes all the keys of attr-map from `ns-meta`.

And yes, it does have a few tests (that's how I found it)

Reply:
Got it, thanks!

What if instead of merging the two metas, we provide the new one as a separate k-v entry and leave it to consumers to merge?

Seems cleaner

Reply:
Done

Reply:
Perfect! Remember to undo this then

Reply:
Ah, yeah, I forgot. One moment